packages/common/src/components/AppHeader/AppHeaderTabs.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { UserTab } from '../../types/utils'
 import styled from 'styled-components'
 import { useHistory, useRouteMatch } from 'react-router-dom'
 import NumberCircle from '../utils/NumberCircle'
 import classNames from 'classnames'
+import { useKeyPress } from '../../utils/userFunctionUtils'
+import { resetPatientState } from '@frontend/webcert/src/store/patient/patientActions'
+import { useDispatch } from 'react-redux'
+import { resetListState } from '@frontend/webcert/src/store/list/listActions'
 
 const Wrapper = styled.nav`
   button {
     display: inline;
     margin-right: 48px;
   }
+
+  .ic-topnav__link {
```

Comment:
We should couple our styles to the css framework, it's better to create our own class name or style component